packages/test/src/activities/cancellable-fetch.ts

```diff
@@ -1,8 +1,9 @@
 import fetch from 'node-fetch';
+import type { AbortSignal as FetchAbortSignal } from 'node-fetch/externals';
 import { Context } from '@temporalio/activity';
 
 export async function cancellableFetch(url: string): Promise<Uint8Array> {
-  const response = await fetch(url, { signal: Context.current().cancellationSignal });
+  const response = await fetch(url, { signal: Context.current().cancellationSignal as FetchAbortSignal });
```

Comment:
Sucks that we have to do this

Reply:
I agree, but node-fetch intentionally redefine the `AbortSignal` interface to also include DOM WG's methods, so passing a standard Node 18 `AbortSignal` won't compile. We could technically go their way and redefine the `AbortSignal` interface to include these extra methods, but then we risk creating even more incompatibilities. I think this cast is the least bad option.